src/bot/mod.rs

```diff
@@ -274,14 +275,15 @@ async fn handle_message(
     client: Client,
     dispatcher: Arc<DownloadDispatcher>,
     whitelist: Arc<Mutex<whitelist::Whitelist>>,
+    superusers: Arc<HashSet<UserId>>,
 ) -> Result<()> {
     let chat = message.chat();
     debug!("Got message from {:?}", chat.id());
     if !matches!(chat, Chat::User(_)) {
         info!("Ignoring message not from private chat ({:?})", chat);
     }
 
-    if chat.id() != SUPERUSER && !whitelist.lock().await.contains(chat.id()) {
+    if !superusers.contains(&chat.id()) && !whitelist.lock().await.contains(chat.id()) {
```

Comment:
Maybe explicitly add all the superusers to the whitelist? Then you wouldn't need to check it here

Reply:
it seems to introduce extra effort of making sure all superusers are within the whitelist (i.e. keeping the list consistent);

Reply:
Ok, what I meant is actually writing the code to add all the superusers to the whitelist at the start of the app, not to make the user do that